content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,44 @@ template = "book-section.html"
 page_template = "book-section.html"
```

Comment:
Quick heads up that I'm going to be a diva about this section :) It is a lot like the home page of the website and I have lots of opinions about what that should look like :smile:

Reply:
I think a couple of iterations here is probably good enough for now, but theres a good chance that I'll revisit / rework this.